lib/include/openamp/remoteproc_virtio.h

```diff
@@ -126,6 +134,20 @@ int rproc_virtio_notified(struct virtio_device *vdev, uint32_t notifyid);
  */
 void rproc_virtio_wait_remote_ready(struct virtio_device *vdev);
 
+/**
+ * @brief Set the remoteproc virtio wait notified function.
+ *
+ * This \wait_notified_cb function will be called to customize the wait, when
```

Comment:
Sorry I did not give you the right method to reference a function in Doxygen format
`\ref wait_notified_cb`

Reply:
Done